app/main/stats.tsx

```
@@ -1,9 +1,188 @@
+        // setHistory(historyRes.data);
+      } catch (err) {
+        console.error('Failed to fetch lobby data', err);
+      }
+    };
+
+    fetchData();
+  }, [token]);
+
+  // Floating pieces
+  const floatingPieces = Array.from({ length: 6 }).map((_, i) => ({
+    translateY: useSharedValue(0),
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)